test/src/cookies.spec.ts

```diff
@@ -834,4 +834,62 @@ describe('Cookie specs', () => {
       expect(await page.cookies()).toHaveLength(0);
     });
   });
+  describe('BrowserContext.cookies', () => {
```

Comment:
let's add tests covering deletion of cookies.

Reply:
added